packages/cactus-plugin-satp-hermes/run-gateway.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
please make this a package.json script and remove this file

Reply:
done!